src/css/Contact.css

```diff
@@ -0,0 +1,23 @@
+.reach{
+padding-top: 1%;
+padding-left: 20%;
+width: 35%;
+height: 23%;
+text-align: left;
+font: 75px/80px Arial;
+letter-spacing: 0;
+color: #FF4081;
+opacity: 1;
+}
+
+.number {
+padding-top: 10%;
+padding-left: 50%;
+width: 80%;
+height: 24%;
+text-align: left;
+font: 40px/50px Arial;
```

Comment:
Change this to font: Regular 75px/50px Whyte Inktrap;

Reply:
uh that looks way to huge on my screen. Is there any way I can make it dynamic

Reply:
Hmm try working with this: "CSS3 have some new values for sizing things relative to the current viewport size: vw, vh, and vmin"

vw is viewport width and wh is viewport height. 

So like font-size: 3vh;

Reply:
Although can you send a screenshot of how it looks with the references rules applied?

Reply:
More viewport stuff: https://css-tricks.com/fun-viewport-units/